diff -ruNp kodi-1dd8613.orig/xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp kodi-1dd8613/xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp
--- kodi-1dd8613.orig/xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp	2017-01-22 03:20:49.000000000 +0300
+++ kodi-1dd8613/xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp	2017-01-22 03:29:56.977838535 +0300
@@ -49,6 +49,9 @@
 #if defined(TARGET_DARWIN_IOS)
 #include "HwDecRender/RendererVTBGLES.h"
 #endif
+#if defined(HAS_CEDARX)
+#include "HwDecRender/RendererSunxi.h"
+#endif
 #if defined(HAS_IMXVPU)
 #include "HwDecRender/RendererIMX.h"
 #endif
diff -ruNp kodi-1dd8613.orig/project/cmake/CMakeLists.txt kodi-1dd8613/project/cmake/CMakeLists.txt
--- kodi-1dd8613.orig/project/cmake/CMakeLists.txt	2017-01-22 19:22:33.000000000 +0300
+++ kodi-1dd8613/project/cmake/CMakeLists.txt	2017-01-23 01:53:46.400465698 +0300
@@ -55,6 +55,7 @@ if(UNIX)
   if(CORE_SYSTEM_NAME STREQUAL linux OR CORE_SYSTEM_NAME STREQUAL freebsd)
     option(ENABLE_X11         "Enable X11 support?" ON)
     option(ENABLE_AML         "Enable AML?" OFF)
+    option(ENABLE_CEDARX      "Enable CEDARX?" OFF)
     option(ENABLE_IMXVPU      "Enable IMX VPU support?" OFF)
   endif()
 endif()
