diff -ruNp kodi-1dd8613.orig/system/keymaps/keyboard.xml kodi-1dd8613/system/keymaps/keyboard.xml
--- kodi-1dd8613.orig/system/keymaps/keyboard.xml	2016-12-04 01:26:17.000000000 +0300
+++ kodi-1dd8613/system/keymaps/keyboard.xml	2017-01-19 23:49:33.137280008 +0300
@@ -94,7 +94,7 @@
       <backslash>ToggleFullScreen</backslash>
       <home>FirstPage</home>
       <end>LastPage</end>
-      <power>ActivateWindow(ShutdownMenu)</power>
+      <power>XBMC.Powerdown()</power>
       <sleep>ActivateWindow(ShutdownMenu)</sleep>
       <!-- PVR windows -->
       <e>ActivateWindow(TVGuide)</e>
diff -ruNp kodi-1dd8613.orig/xbmc/powermanagement/linux/LogindUPowerSyscall.cpp kodi-1dd8613/xbmc/powermanagement/linux/LogindUPowerSyscall.cpp
--- kodi-1dd8613.orig/xbmc/powermanagement/linux/LogindUPowerSyscall.cpp	2016-12-04 01:26:17.000000000 +0300
+++ kodi-1dd8613/xbmc/powermanagement/linux/LogindUPowerSyscall.cpp	2017-01-19 23:47:03.615058549 +0300
@@ -53,7 +53,7 @@ CLogindUPowerSyscall::CLogindUPowerSysca
   m_canPowerdown = LogindCheckCapability("CanPowerOff");
   m_canReboot    = LogindCheckCapability("CanReboot");
   m_canHibernate = LogindCheckCapability("CanHibernate");
-  m_canSuspend   = LogindCheckCapability("CanSuspend");
+  m_canSuspend   = false;
 
   InhibitDelayLock();
 
@@ -98,7 +98,7 @@ CLogindUPowerSyscall::~CLogindUPowerSysc
 
 bool CLogindUPowerSyscall::Powerdown()
 {
-  return LogindSetPowerState("PowerOff");
+  return Suspend();
 }
 
 bool CLogindUPowerSyscall::Reboot()
